src/test/java/api/support/matchers/TextDateTimeMatcher.java

```diff
@@ -3,11 +3,16 @@
 import org.hamcrest.Description;
 import org.hamcrest.Matcher;
 import org.hamcrest.TypeSafeMatcher;
+import org.hamcrest.core.IsNull;
 import org.joda.time.DateTime;
 import org.joda.time.Seconds;
 
 public class TextDateTimeMatcher {
   public static Matcher<String> isEquivalentTo(DateTime expected) {
+    if (expected == null) {
```

Comment:
Does this mean that if the expected value is null, the matcher passes?

I would think that cases where the property is intended to be not present could be checked more explicitly with existing matchers.

It seems odd to me to hide it here, and could result in subtle bugs in the tests (if an accidentally null variable is used for example).

Reply:
Yes, in case when matching values is also null. I got rid of this check excluding the mathing value which can be null (hold expiration date is null when the item hasn't been checked in).